include/SFML/System/Time.inl

```diff
@@ -213,48 +213,55 @@ constexpr Time& operator*=(Time& left, std::int64_t right)
 ////////////////////////////////////////////////////////////
 constexpr Time operator/(Time left, float right)
 {
+    assert(right != 0 && "Cannot divide by zero");
     return seconds(left.asSeconds() / right);
 }
 
 
 ////////////////////////////////////////////////////////////
 constexpr Time operator/(Time left, std::int64_t right)
 {
+    assert(right != 0 && "Cannot divide by zero");
     return microseconds(left.asMicroseconds() / right);
 }
 
 
 ////////////////////////////////////////////////////////////
 constexpr Time& operator/=(Time& left, float right)
 {
+    assert(right != 0 && "Cannot divide by zero");
     return left = left / right;
 }
 
 
 ////////////////////////////////////////////////////////////
 constexpr Time& operator/=(Time& left, std::int64_t right)
 {
+    assert(right != 0 && "Cannot divide by zero");
     return left = left / right;
 }
 
 
 ////////////////////////////////////////////////////////////
 constexpr float operator/(Time left, Time right)
 {
+    assert(right.asMicroseconds() != 0 && "Cannot divide by zero");
     return left.asSeconds() / right.asSeconds();
 }
 
 
 ////////////////////////////////////////////////////////////
 constexpr Time operator%(Time left, Time right)
 {
+    assert(right.asMicroseconds() != 0 && "Cannot divide by zero");
```

Comment:
Maybe "modulo by zero"? 

Even though closely related to division, modulo is a separate operation, and I imagine that people who get "division by zero" errors would look for divisions, not modulos.